terraform/gitops/generate-files/templates/pm4ml/rbac.yaml.tpl

```diff
@@ -26,4 +26,16 @@ spec:
           "subject_id": "{{ print .Subject }}"
         }
   mutators:
-    - handler: header
\ No newline at end of file
+    - handler: header
+---
+apiVersion: mojaloop.io/v1
+kind: MojaloopRole
+metadata:
+  name: pm4mladmin
+  namespace: ory
```

Comment:
Can you parameterise the ory name space value?

Reply:
done